scipy/ndimage/src/ni_measure.c

```diff
@@ -205,7 +205,7 @@ typedef struct {
     npy_intp index;
     void *next, *prev;
     npy_uint16 done;
-    npy_uint8 cost;
+    npy_uint16 cost;
```

Comment:
It looks like a simple enough fix, but I will reiterate the concerns in the matching issue about providing a concise unit/regression test to go along with it if possible.

It seems there were some recent improvements to the reproducing examples there so maybe they could be converted to a test?

Reply:
> It seems there were some recent improvements to the reproducing examples there so maybe they could be converted to a test?

Yes, I can add a regression test with a 2x2 matrix tomorrow.

Reply:
I added a simple regression test.